examples/flutter_gallery/lib/demo/buttons_demo.dart

```diff
@@ -246,4 +257,21 @@ class _ButtonsDemoState extends State<ButtonsDemo> {
       ),
     );
   }
+
+  Widget buildIosButton() {
+    return new Align(
+      alignment: const FractionalOffset(0.5, 0.4),
+      child: new ButtonBar(
+        mainAxisSize: MainAxisSize.min,
+        children: <Widget>[
+          new CupertinoButton(
+            text: 'iOS Button',
+            onPressed: () {
+              // Perform some action
```

Comment:
The existing demo doesn't do anything on press. Might be a bit annoying if things kept popping up on each press for each tap vs a dialog result etc